src/Selenium2Library/keywords/_selectelement.py

```diff
@@ -8,15 +8,24 @@ class _SelectElementKeywords(KeywordGroup):
 
     # Public
 
-    def get_list_items(self, locator):
-        """Returns the values in the select list identified by `locator`.
+    def get_list_items(self, locator, label=True):
+        """Returns the labels or values in the select list identified by `locator`.
 
         Select list keywords work on both lists and combo boxes. Key attributes for
         select lists are `id` and `name`. See `introduction` for details about
         locating elements.
+        
+        Sample:
+        | ${labels1} = | Get List Items | xpath=//h1 |
+        | ${labels2} = | Get List Items | xpath=//h1 | label=${True} |
+        | ${values} = | Get List Items | xpath=//h1 | label=${False} |
+        | Should Be Equal | ${labels1} | ${labels2} |
         """
         select, options = self._get_select_list_options(locator)
-        return self._get_labels_for_options(options)
+        if label:
```

Comment:
I have changed to label=False, but it is true. maybe should fix the issue about handling Boolean arguments.